app/build.gradle.kts

```diff
@@ -73,7 +73,7 @@ android {
             )
         }
         debug {
-            isMinifyEnabled = true
+            isMinifyEnabled = false
```

Comment:
Not sure why or when this got set to `true`. Disabled as I don't need minification slowing down debug builds.